api/docs/test_suite.dox

```diff
@@ -315,6 +315,15 @@ The comments at the top of runsuite_ssh.cmake describe additional options.
 
 Unfortunately our test suite is not as clean as it could be.  Some tests can be flaky and while they pass on the machines of the existing developers and on our automated test machines, they may fail occasionally on a new machine.  Please search the issue tracker before filing a new issue to see if a test failure has already been seen once before.  We welcome contributions to fix flaky tests.
 
+Flaky tests are marked in one of two ways:
```

Comment:
Let's update the runsuite.cmake comments (line 141) about _FLAKY as well as they are stale.

Reply:
What's the right wording to use here?

The code refers to #2974 but that seems to be just a general bug for recording the addition of aarch64 support.

Here's the best I can come up with:
```
Skip tests marked _FLAKY. This is one of the ways that flaky tests are marked as such
(the other way, for higher priority flakes to fix, is to mention them in runsuite_wrapper.pl).
We need CDash to ignore flakes and avoid going red and sending emails,
and we rely on our CI for a history of _FLAKY results.
```

Reply:
We can remove the CDash sentence as we no longer use CDash.

Reply:
Done